packages/vscode-extension/README.md

```diff
@@ -1,64 +1,85 @@
+SuiteCloud Extension for Visual Studio Code is part of the SuiteCloud Software Development Kit (SuiteCloud SDK), a set of tools to customize your NetSuite accounts. The SuiteCloud Extension for Visual Studio Code allows you to customize your SuiteCloud Development Framework (SDF) projects for NetSuite. The available SDF projects include **SuiteApp projects** — self-contained, standalone projects that you can publish in the SuiteApp Marketplace for other users to download and install, and **Account Customization Projects** — intended for customizations on accounts you own, such as your production, development, and sandbox accounts.
 
-> 💡 To read more about SDF, visit [NetSuite's Help Center.](https://system.netsuite.com/app/help/helpcenter.nl?fid=section_4702656040.html)
+Read more about SuiteCloud SDK and SDF in <a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=chapter_156026236161.html" rel="noopener noreferrer" target=blank>Getting Started with SuiteCloud SDK</a> and <a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_4702656040.html" rel="noopener noreferrer" target=blank>SuiteCloud Development Framework Architecture</a>.
 
-The following types of SDF projects are available:
-- SuiteApp projects — self-contained, standalone projects that you can publish in the SuiteApp Marketplace for other users to download and install.
-  > 💡 To read more about SuiteApp projects, visit [NetSuite's Help Center.](https://system.netsuite.com/app/help/helpcenter.nl?fid=subsect_1509931104.html)
-- Account customization projects — intended for customizations on accounts you own, such as your production, development, and sandbox accounts.
-  > 💡 To read more about account customization projects, visit [NetSuite's Help Center.](https://system.netsuite.com/app/help/helpcenter.nl?fid=subsect_1510680449.html)
+## <font size="6"> <span style="color:gray"> Installation prerequisites </font> </span>
+The following software is required to work with SuiteCloud Extension for Visual Studio Code:
+- Node.js version 14.16.0 LTS
+- Oracle JDK version 11
 
-![Account customization project](resources/acpMain.png)
+Read the detailed list of prerequisites in <a href="https://system.netsuite.com/app/help/helpcenter.nl?fid=section_159223197655.html" rel="noopener noreferrer" target=blank>Installation Requirements for SuiteCloud Extension for Visual Studio Code</a>.
 
-SuiteCloud extension for Visual Studio Code is part of SuiteCloud Software Development Kit (SuiteCloud SDK), a set of tools to customize NetSuite accounts.
+## <font size="6"> <span style="color:gray"> Installing SuiteCloud Extension for Visual Studio Code </font> </span>
+To install SuiteCloud Extension for Visual Studio Code, follow these steps:
 
-💡 To read more about SuiteCloud SDK, visit [NetSuite's Help Center.](https://system.netsuite.com/app/help/helpcenter.nl?fid=chapter_156026236161.html)
+1. Launch Visual Studio Code and select the **Extensions** tab.
```

Comment:
Not sure if Extensions tab is clear here. in mac, you have to go to Code -> Preferences -> Extensions.
From the Microsoft documention:
'Bring up the Extensions view by clicking on the Extensions icon in the Activity Bar on the side of VS Code or the View: Extensions command (⇧⌘X).'
Can we use something like this?
https://code.visualstudio.com/docs/editor/extension-marketplace